sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ClientSideRequestStatistics.java

```diff
@@ -377,5 +388,9 @@ public String getRequestCharge() {
         public RequestTimeline getRequestTimeline() {
             return requestTimeline;
         }
+
+        public Double getBackendLatencyInMs() {
```

Comment:
in case we are hitting GW, does GW passes the value returned by BackEnd? or does it add its own additional latency to the header?

Reply:
AFAIK, gateway passes whatever backend has passed it, but I am confirming it with backend folks. Will update this conversation once I hear back from them.

Reply:
Please ask this question as well - What if Gateway do some retry , does this value will be the e2e time from gateway or  last response from BE, also in case of Strong consistency which response gateway will consider

Reply:
Confirmed with Ajay Parulekar from backend team, gateway will pass the same value to client in all cases. This is just backend latency.

Reply:
@simplynaveen20 @moderakh  - what I got from backend folks is that read for every replica will have its own `x-ms-request-duration` value. So it will have backend latency for multiple replica reads.

Reply:
Thats what gateway will give only one response after consuming multiple from back end, so this time will be misleading. Lets talk offline